Library/Formula/duck.rb

```diff
@@ -0,0 +1,29 @@
+class Duck < Formula
+  homepage "https://duck.sh/"
+  url "https://dist.duck.sh/duck-src-16336.tar.gz"
+  version "4.6.2.16336"
+  head "https://svn.cyberduck.io/trunk/"
+  sha1 "377f8ca3d111416a20122e0e5f41f189c5eda99e"
+
+  depends_on :java => ["1.7", :build]
+  depends_on :xcode => :build
+  depends_on "ant" => :build
+  depends_on "openssl"
+
+  def install
+    system "ant", "-Dbuild.compile.target=1.7", "-Drevision=16336", "cli"
+    system "install_name_tool", "-change", "/usr/lib/libcrypto.0.9.8.dylib", "/usr/local/opt/openssl/lib/libcrypto.dylib", "build/duck.bundle/Contents/Frameworks/libPrime.dylib"
```

Comment:
I am hesitant to change this in our upstream build just now but could schedule it on our roadmap for future consideration. Not sure how one can override linker dependencies defined in the project.